ratpack-pac4j/src/main/java/ratpack/pac4j/internal/Pac4jAuthenticator.java

```diff
@@ -81,13 +82,28 @@ public void handle(Context ctx) throws Exception {
   }
 
   public Clients createClients(Context ctx, PathBinding pathBinding) throws Exception {
+    String callback = createCallbackUrl(ctx.get(PublicAddress.class), pathBinding, path);
+
+    return new Clients(callback, getClientsFromProvider(ctx, clientsProvider));
+  }
+
+  private String createCallbackUrl(PublicAddress publicAddress, PathBinding pathBinding, String path) {
     String boundTo = pathBinding.getBoundTo();
-    PublicAddress publicAddress = ctx.get(PublicAddress.class);
-    String absoluteCallbackUrl = publicAddress.get() + boundTo + "/" + path;
+    URI address = publicAddress.get();
+
+    if(!boundTo.isEmpty()) {
+      if (!(address.toString().endsWith("/") || boundTo.startsWith("/"))) {
```

Comment:
This condition is unnecessary because according to the contracts public address should not end with a `/` and bound path binding should not start with a `/`